docs/source/getting-started/quickstart.rst

```diff
@@ -164,8 +164,17 @@ For uploading files to the cluster, see :ref:`Syncing Code and Artifacts <sync-c
 Stop/terminate a cluster
 =========================
 
-When you are done, run :code:`sky stop mycluster` to stop the cluster. To
-terminate a cluster instead, run :code:`sky down mycluster`.
+When you are done, stop the cluster with:
```

Comment:
nit: add ``sky stop`` here as well, same as the launch example above? Personally thought this way it's more noticeable.

Reply:
Added!